appinventor/components/src/com/google/appinventor/components/runtime/ChartDataBase.java

```diff
@@ -414,6 +413,44 @@ public void run() {
         });
     }
 
+    /**
+     * Imports data from the specified CloudDB component with the provided tag identifier.
+     *
+     * @param cloudDB  CloudDB component to import from
+     * @param tag  the identifier of the value to import
+     */
+    @SimpleFunction(description = "Imports data from the specified CloudDB component, given the tag of the " +
+        "value to use. The value is expected to be a YailList consisting of entries compatible with the " +
+        "Data component.")
+    public void ImportFromCloudDB(final CloudDB cloudDB, final String tag) {
+        // Get the Future YailList object from the CloudDB data
+        final Future<List> list = cloudDB.getDataValue(tag);
```

Comment:
Seems to cause out of order importing:
![image](https://user-images.githubusercontent.com/34982762/62228983-02e1f900-b3bf-11e9-93fd-e73ad160a038.png)

Reply:
Issue unrelated to changes of this PR. See #1752